src/api/routes/samples.js

```diff
@@ -2,31 +2,50 @@ const SamplesService = require('../route-services/samples');
 
 const samplesService = new SamplesService();
 
+const {
+  expressAuthorizationMiddleware,
+} = require('../../utils/authMiddlewares');
+
 module.exports = {
-  'samples#get': (req, res, next) => {
-    samplesService.getSamples(req.params.projectUuid)
-      .then((data) => res.json(data))
-      .catch(next);
-  },
-  'samples#getSamplesByExperimentId': (req, res, next) => {
-    samplesService.getSamplesByExperimentId(req.params.experimentId)
-      .then((data) => res.json(data))
-      .catch(next);
-  },
-  'samples#update': (req, res, next) => {
-    const { body, params: { projectUuid } } = req;
+  'samples#get': [
+    expressAuthorizationMiddleware,
+    (req, res, next) => {
+      samplesService.getSamples(req.params.projectUuid)
+        .then((data) => res.json(data))
+        .catch(next);
+    }],
+  'samples#getSamplesByExperimentId': [
+    expressAuthorizationMiddleware, (req, res, next) => {
+      samplesService.getSamplesByExperimentId(req.params.experimentId)
+        .then((data) => res.json(data))
+        .catch(next);
+    }],
+  'samples#update': [
```

Comment:
Check if this endpint is stil lbeing used

Reply:
probably used for internal sampels proeprties updates

Reply:
Yea this is still used in the UI by `saveSamples`.